js/dataframe/shared/Table.svelte

```diff
@@ -529,7 +529,7 @@
 
 	function set_cell_widths(): void {
 		const widths = cells.map((el, i) => {
-			return el?.clientWidth || 0;
+			return el?.offsetHeight || 0;
```

Comment:
Why are we measuring the height to get the width of the cell?

Reply:
Whoops my bad, will fix